ColorPatterns/Scenes/Game/GameController.swift

```diff
@@ -42,33 +38,34 @@ final class GameController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         updateColorBars()
-        setupSwipeDownGesture(direction: .down)
-        setupGameTimer()
-        setupViewModelDelegate()
-        setupScoreLabel()
+        viewModel.startTimer()
     }
 
     @IBAction private func colorBarPressed(_ sender: UIButton) {
+        // TODO: Try not to use tags, make an another way to identify buttons
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c279b1ccceb81114848aa1e590426203038db92d/ColorPatterns/Scenes/Game/GameController.swift#L45"></span>TODOs should be resolved (Try not to use tags, make an a…).<br />
<code>todo</code> <code>GameController.swift:45</code></td>
    </tr>
  </tbody>
</table>